certification_organism/data/product_category.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
```

Comment:
Why do you export this? You should remove this file, as it tries to set data of other module with too many values in it, and don't change any values